presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/PermissionMetadata.java

```diff
@@ -56,6 +56,6 @@ public boolean isGrantOption()
 
     public HivePrivilegeInfo toHivePrivilegeInfo()
     {
-        return new HivePrivilegeInfo(permission, grantOption, new HivePrincipal(USER, "admin"), new HivePrincipal(USER, "admin"));
+        return new HivePrivilegeInfo(permission, grantOption, new HivePrincipal(USER, "<unknown>"), new HivePrincipal(USER, "<unknown>"));
```

Comment:
I don't see a benefit in this change.
File metastore hard-codes a bunch of behaviors
https://github.com/prestosql/presto/blob/5c991a59ff2e7f78da76fda2f2e9dc4096cca092/presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastore.java#L118-L123

and this is just one such hard-coded behavior.

Reply:
I think file metastore should behave in the same like regular HMS. IMO it would be best if we could remove every hard-coded behaviour.

I think this change makes it clear what is still missing in current implementation.

Reply:
@findepi ping